doc/mkdocs/docs/api/basic_json/at.md

```diff
@@ -134,7 +152,7 @@ Strong exception safety: if an exception occurs, the original value stays intact
     --8<-- "examples/at__object_t_key_type_const.output"
     ```
 
-??? example "Example (3) access specified element via JSON Pointer"
+??? example "Example (4) access specified element via JSON Pointer"
```

Comment:
Speaking of examples earlier … this is where I'd really need to add some. If not in this PR then soon after.